dev-docs/source/ReleaseChecklist.rst

```diff
@@ -177,7 +194,11 @@ Beta Testing Ends
 Just before release
 ###################
 
-* As one of the final steps in preparing to tag the release, remove all separate release note files and sub-file structure to leave just the upper level release notes e.g. diffraction.rst, index.rst, framework.rst etc.
+* As one of the final steps in preparing to tag the release:
+
+  * Add any final release notes manually to the main release note pages.
+  * Remove unused headings from the main release note pages.
+  * Remove all separate release note files and sub-file structure to leave just the main release note pages.
 
```

Comment:
I think its worth adding a step/section about checking the main branch for release notes using the new release note finder tool.